src/pages/donate.astro

```diff
@@ -0,0 +1,165 @@
+        </p>
+        <p class="font-light">
+          Thank you!
+        </p>
+        <button type="button">
+          <a href="https://opencollective.com/nixos#support">
+            Contribute
+          </a>
+        </button>
+      </div>
+      <div class="flex flex-col gap-4 justify-center">
+        <h2 class="text-4xl font-bold text-nixdarkblue pb-2">
+          Donate with SEPA bank transfer
+        </h2>
+        <p class="font-light">
+          The NixOS Foundation also accepts donations via SEPA bank transfer. Please send payments to:
+        </p>
+        <pre class="font-mono bg-nixlighterblue text-nixdarkerblue font-bold whitespace-pre-line p-8 rounded-3xl">
+Stichting NixOS Foundation, Utrecht, The Netherlands
+
+IBAN: NL13 BUNQ 2061 2530 32
+BIC:  BUNQNL2AXXX</pre>
+        <p class="text-xs font-light">
+          Please add your name and purpose (donation) in the description of the transfer.
+          <i>
+            If you would like to support us with amounts above 2000 euro per year, please contact us at
+            <a href="mailto:foundation@nixos.org">foundation@nixos.org</a>
+            , as this will have tax implications.
+          </i>
+        </p>
+      </div>
+    </Container>
+
+    <img src="/divider/slope_down.svg" class="w-full object-fill object-[60%_top] h-16 -scale-y-100" />
+    <div class="bg-nixdarkblue py-16 text-white">
+      <Container class="grid md:grid-cols-2 gap-8 px-12">
+        <div class="flex flex-col justify-center">
+          <div class="grid grid-cols-2 sm:grid-cols-5 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4">
+            <ExternalLogo class="p-5" href="https://www.4tu.nl/" src="/images/logos/4tu.svg" alt="4TU logo" />
+            <ExternalLogo class="" href="https://www.antithesis.com/" src="/images/logos/antithesis.svg" alt="Antithesis logo" />
+            <ExternalLogo class="p-5" href="https://deploy.equinix.com/" src="/images/logos/equinix_metal.svg" alt="Equinix logo" />
+            <ExternalLogo class="p-5" href="https://www.fastly.com/" src="/images/logos/fastly.svg" alt="Fastly logo" />
+            <ExternalLogo class="" href="https://www.hetzner.com/" src="/images/logos/hetzner.png" alt="Hetzner logo" />
+            <ExternalLogo class="p-5" href="https://www.logicblox.com" src="/images/logos/logicblox.png" alt="LogicBlox logo" />
+            <ExternalLogo class="p-3" href="https://www.macstadium.com/" src="/images/logos/macstadium.png" alt="MacStadium logo" />
+            <ExternalLogo class="p-3" href="https://github.com/snabbco/snabb" src="/images/logos/snabb.png" alt="Snabb logo" />
+            <ExternalLogo class="" href="https://www.tudelft.nl/" src="/images/logos/tudelft.png" alt="TU Delft logo" />
+            <ExternalLogo class="p-0" href="https://www.uu.nl/" src="/images/logos/universiteit_utrecht.png" alt="Universiteit Utrecht logo" />
```

Comment:
would be great if the data could be moved into the `content` directory like it has been done with the menus and iterated over here

Reply:
You can find a reference for how it's done here:
- https://github.com/NixOS/nixos-homepage/blob/main/src/content/menus/header.yaml
- https://github.com/NixOS/nixos-homepage/blob/main/src/layouts/Header.astro

in the Header Layout this is basically done via the import:
```typescript
const headerMenu = await getEntry('menus', 'header');
```
and the iteration over the imported data
```typescript
{headerMenu.data.items.map((item) => (
  <a class={`pt-8 pb-6 ${navbarItemStyle}`} href={item.link}>{item.name}</a>
))}
```

Of course there is also some reference in the [Astro docs](https://docs.astro.build/en/guides/content-collections/)

Reply:
Feel free to include the padding property into your content, but keep in mind you can't split up the padding class into prefix and value https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Reply:
Done!